### ISSUE TITLE
inode 100% used on google drive rclone mount

### ISSUE THREAD

**Issue by milindpatel63** [rclone/rclone#6965]
#### What is the problem you are having with rclone?
I have mounted my google drive via rclone mount. I am using netdata to monitor my system, and it's showing 100% inode usage on that mount.
```
df -i /mnt/google-drive
Filesystem         Inodes      IUsed IFree IUse% Mounted on
gdrive:        1000000000 1000000000     0  100% /mnt/google-drive
```
```
sudo rclone --config=/home/user/.config/rclone/rclone.conf about gdrive:
Total:   202G
Used:    79.728G
Free:    24.894G
Trashed: 0
Other:   97.378G
```
```
sudo cat /home/user/.config/rclone/rclone.conf
[gdrive]
type = drive
scope = drive
token = 
```
Here's my systemd unit file that mounts my rclone google drive
```
[Unit]
Description=Google Drive (rclone)
AssertPathIsDirectory=/mnt/google-drive
After=network-online.target

[Service]
Type=simple
ExecStart=/usr/bin/rclone mount \
        --config=/home/user/.config/rclone/rclone.conf \
        --allow-other \
        --cache-tmp-upload-path=/tmp/rclone/upload \
        --cache-chunk-path=/tmp/rclone/chunks \
        --cache-workers=8 \
        --cache-writes \
        --cache-dir=/tmp/rclone/vfs \
        --cache-db-path=/tmp/rclone/db \
        --no-modtime \
        --drive-use-trash \
        --stats=0 \
        --checkers=16 \
        --bwlimit=100M \
        --dir-cache-time=60m \
        --cache-info-age=60m gdrive:/ /mnt/google-drive
ExecStop=/bin/fusermount -u /mnt/google-drive
Restart=always
RestartSec=10

[Install]
WantedBy=default.target
```
#### What is your rclone version (output from `rclone version`)
```
rclone -V
rclone v1.53.3-DEV
- os/arch: linux/amd64
- go version: go1.18.1
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
Pop OS 22.04

#### Which cloud storage system are you using? (e.g. Google Drive)
Google Drive

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
```
/usr/bin/rclone mount \
        --config=/home/user/.config/rclone/rclone.conf \
        --allow-other \
        --cache-tmp-upload-path=/tmp/rclone/upload \
        --cache-chunk-path=/tmp/rclone/chunks \
        --cache-workers=8 \
        --cache-writes \
        --cache-dir=/tmp/rclone/vfs \
        --cache-db-path=/tmp/rclone/db \
        --no-modtime \
        --drive-use-trash \
        --stats=0 \
        --checkers=16 \
        --bwlimit=100M \
        --dir-cache-time=60m \
        --cache-info-age=60m gdrive:/ /mnt/google-drive
```

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
```
sudo rclone --config=/home/user/.config/rclone/rclone.conf -vv copy ~/temp.txt gdrive:temp.txt
2023/04/21 19:42:45 DEBUG : rclone: Version "v1.53.3-DEV" starting with parameters ["rclone" "--config=/home/user/.config/rclone/rclone.conf" "-vv" "copy" "/home/user/temp.txt" "gdrive:temp.txt"]
2023/04/21 19:42:45 DEBUG : Creating backend with remote "/home/user/temp.txt"
2023/04/21 19:42:45 DEBUG : Using config file from "/home/user/.config/rclone/rclone.conf"
2023/04/21 19:42:45 DEBUG : fs cache: adding new entry for parent of "/home/user/temp.txt", "/home/milindpatel63"
2023/04/21 19:42:45 DEBUG : Creating backend with remote "gdrive:temp.txt"
2023/04/21 19:42:45 DEBUG : Google drive root 'temp.txt': root_folder_id = "***" - save this in the config to speed up startup
2023/04/21 19:42:47 DEBUG : temp.txt: Need to transfer - File not found at Destination
2023/04/21 19:42:51 DEBUG : temp.txt: MD5 = 188a0ff1e830261d0253b185a1c0f94f OK
2023/04/21 19:42:51 INFO  : temp.txt: Copied (new)
2023/04/21 19:42:51 INFO  : 
Transferred:   	         5 / 5 Bytes, 100%, 1 Bytes/s, ETA 0s
Transferred:            1 / 1, 100%
Elapsed time:         5.9s

2023/04/21 19:42:51 DEBUG : 4 go routines active
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by animosity22** [comment]
That version is like 8 years old.

Update it.

**Comment by ncw** [comment]
We had problems with this in #2089 and #2356 which should have been fixed by 1.53

If this is still a problem for 1.62 then I'll re-open the issue @milindpatel63 but it works fine here

```
$ df -i /mnt/tmp
Filesystem         Inodes IUsed      IFree IUse% Mounted on
drive:test     1000000000     0 1000000000    0% /mnt/tmp
```

**Comment by milindpatel63** [comment]
I apologize. I was running an older version.
The apt repo in my Pop OS is still outdated.
```
$ apt search rclone
Sorting... Done
Full Text Search... Done
bomberclone/jammy,jammy 0.11.9-7.1 amd64
  free Bomberman clone

bomberclone-data/jammy,jammy,jammy,jammy 0.11.9-7.1 all
  Data files for bomberclone game

git-annex-remote-rclone/jammy,jammy,jammy,jammy 0.6-1 all
  rclone-based git annex special remote

golang-github-rclone-rclone-dev/jammy-security,jammy-security,jammy-updates,jammy-updates 1.53.3-4ubuntu1.22.04.1 all
  go source code of rclone

rclone/jammy-security,jammy-updates,now 1.53.3-4ubuntu1.22.04.1 amd64 [installed,automatic]
  rsync for commercial cloud storage

rclone-browser/jammy,jammy,now 1.8.0-1.2build1 amd64 [installed]
  Simple cross platform GUI for rclone

```
I used the rclone script on it's official website here https://rclone.org/downloads/#script-download-and-install to update to the latest version and this inode issue is fixed now.
